scripts/setup-cgct.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+REPO_URL="https://service.termux-pacman.dev/gpkg-dev"
+
+: "${TERMUX_PKG_TMPDIR:="/tmp"}"
+TMPDIR_CGCT="${TERMUX_PKG_TMPDIR}/cgct"
+
+if [ ! -d "$TMPDIR_CGCT" ]; then
+	mkdir -p "$TMPDIR_CGCT"
+fi
+
+. $(dirname "$(realpath "$0")")/properties.sh
+
+if [ ! -d "$CGCT_DIR" ]; then
+	curl "${REPO_URL}/x86_64/gpkg-dev.json" -o "${TMPDIR_CGCT}/cgct.json"
+	for pkgname in cbt cgt; do
```

Comment:
What does cbt and cgt stand for?

Reply:
cbt - Cross Binutils for Termux (https://github.com/termux-pacman/glibc-packages/blob/main/gpkg-dev/cbt/PKGBUILD)
cgt - Cross GCCs for Termux (https://github.com/termux-pacman/glibc-packages/blob/main/gpkg-dev/cgt/PKGBUILD)